### PR TITLE
update button drive v1.0.1

### PR DESCRIPTION
**Pull request by jiejieTop**
Comments use English comments, style is similar to rtt, button source has been modified, others slightly modified.

### CODE REVIEW

File: peripherals/button/package.json [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
+        "VER_SHA" : "799daaade8ea7be4237aad1dab2b2871e5bd0b8d"},
 
     {
         "version" : "latest",
-        "URL" : "https://github.com/jiejieTop/rtt_btn_drv.git",
-        "filename" : "Null for git package",
-        "VER_SHA" : "master"}
+        "URL" : "https://github.com/jiejieTop/rtpkg_button.git",
+        "filename" : "button-latest",
+        "VER_SHA" : "799daaade8ea7be4237aad1dab2b2871e5bd0b8d"}
```
Comment (armink) [inline peripherals/button/package.json]: latest 的可以不用写 hash 值，写分支名，参考：https://github.com/RT-Thread/packages/blob/master/iot/at_device/package.json#L14

File: peripherals/button/package.json [hunk 2]
```diff
@@ -7,16 +7,16 @@
     ],
     "site" : [
     {
-        "version" : "v1.0.0", 
-        "URL" : "https://github.com/jiejieTop/rtt_btn_drv.git", 
-        "filename" : "button-1.0.0.zip",
-        "VER_SHA" : "0cf219a23f3568b82da1c055e2d415eded3b7230"},
+        "version" : "v1.0.1",
```
Comment (armink) [inline peripherals/button/package.json]: 已发布版本建议不要删除了吧。另外，建议更新前，先去 GitHub release 一下。

参考：https://github.com/RT-Thread-packages/at_device/releases

File: peripherals/button/Kconfig [hunk 3]
```diff
@@ -54,20 +54,24 @@ if PKG_USING_BUTTON
...
         help
             Select the package version
 
-        config PKG_USING_BUTTON_V100
-            bool "v1.0.0"
-
+        config PKG_USING_BUTTON_V101
+            bool "v1.0.1"
+			
+		config PKG_USING_BUTTON_V100
```
Comment (BernardXiong) [inline peripherals/button/Kconfig]: 请做个对齐

File: peripherals/button/Kconfig [hunk 4]
```diff
@@ -54,20 +54,24 @@ if PKG_USING_BUTTON
...
+					
         config PKG_USING_BUTTON_LATEST_VERSION
             bool "latest"
     endchoice
           
     config PKG_BUTTON_VER
        string
-       default "v1.0.0"    if PKG_USING_BUTTON_V100
+       default "v1.0.1"    if PKG_USING_BUTTON_V101
+	   default "v1.0.0"    if PKG_USING_BUTTON_V100
```
Comment (BernardXiong) [inline peripherals/button/Kconfig]: 同对齐

File: peripherals/button/Kconfig (hunk 3 above)
Comment (jiejieTop) [reply peripherals/button/Kconfig]: ok

File: peripherals/button/package.json (hunk 2 above)
Comment (jiejieTop) [reply peripherals/button/package.json]: ok

File: peripherals/button/package.json (hunk 1 above)
Comment (jiejieTop) [reply peripherals/button/package.json]: ok

**Pull request merged**